test/fixed_rank.jl

```diff
@@ -9,69 +9,36 @@ include("utils.jl")
     v = UMVTVector([0.0 0.0; 0.0 0.0; 1.0 1.0], [1.0 0.0; 0.0 1.0], zeros(2, 2))
     @test repr(M) == "FixedRankMatrices(3, 2, 2, ℝ)"
     @test repr(Mc) == "FixedRankMatrices(3, 2, 2, ℂ)"
-    if VERSION >= v"1.6.0-DEV.772"
-        @test sprint(show, "text/plain", x) == """
-        SVDMPoint{Matrix{Float64}, Vector{Float64}, Matrix{Float64}}
-        U factor:
-         3×2 Matrix{Float64}:
-          1.0  0.0
-          0.0  1.0
-          0.0  0.0
-        singular values:
-         2-element Vector{Float64}:
-          1.0
-          1.0
-        Vt factor:
-         2×2 Matrix{Float64}:
-          1.0  0.0
-          0.0  1.0"""
-        @test sprint(show, "text/plain", v) == """
-        UMVTVector{Matrix{Float64}, Matrix{Float64}, Matrix{Float64}}
-        U factor:
-         3×2 Matrix{Float64}:
-          0.0  0.0
-          0.0  0.0
-          1.0  1.0
-        M factor:
-         2×2 Matrix{Float64}:
-          1.0  0.0
-          0.0  1.0
-        Vt factor:
-         2×2 Matrix{Float64}:
-          0.0  0.0
-          0.0  0.0"""
-    else
-        @test sprint(show, "text/plain", x) == """
-        SVDMPoint{Array{Float64,2},Array{Float64,1},Array{Float64,2}}
-        U factor:
-         3×2 Array{Float64,2}:
-          1.0  0.0
-          0.0  1.0
-          0.0  0.0
-        singular values:
-         2-element Array{Float64,1}:
-          1.0
-          1.0
-        Vt factor:
-         2×2 Array{Float64,2}:
-          1.0  0.0
-          0.0  1.0"""
-        @test sprint(show, "text/plain", v) == """
-        UMVTVector{Array{Float64,2},Array{Float64,2},Array{Float64,2}}
-        U factor:
-         3×2 Array{Float64,2}:
-          0.0  0.0
-          0.0  0.0
-          1.0  1.0
-        M factor:
-         2×2 Array{Float64,2}:
-          1.0  0.0
-          0.0  1.0
-        Vt factor:
-         2×2 Array{Float64,2}:
-          0.0  0.0
-          0.0  0.0"""
-    end
+    @test sprint(show, "text/plain", x) == """
+    $(sprint(show, SVDMPoint{Matrix{Float64}, Vector{Float64}, Matrix{Float64}}))
+    U factor:
+     3×2 $(sprint(show, Matrix{Float64})):
```

Comment:
You could also use the print of `x.U` for the whole block lines 15--18?

Reply:
OK, I'll try that.

Reply:
I don't how to print `x.U` to get the correct spacing :slightly_frowning_face: .

Reply:
Ah, I see, because we add an indent (1 or 2 spaces?), see https://github.com/JuliaManifolds/Manifolds.jl/blob/67ee0f103338ec205017fa4d9b448b2c1b22874f/test/vector_bundle.jl#L121
on a place where I tricked something like that, too.

Reply:
I think we can do that in another PR.

Reply:
Ok.